worldengine/cli/main.py

```diff
@@ -391,8 +391,8 @@ def main():
     if args.seed is not None:
         seed = int(args.seed)
     else:
-        seed = random.randint(0, 65535)#RNG initialization is done automatically
-    random.seed(seed)
+        seed = numpy.random.randint(0, 999999)  # RNG initialization is done automatically
```

Comment:
If necessary, you can always use modulus (%) 65536 which will always give you a reliable 2^16 for Platec for a given seed that happens to be bigger.